bluepyemodel/evaluation/fitness_calculator_configuration.py

```diff
@@ -105,24 +105,55 @@ def __init__(
                 .. code-block::
 
                     [
-                        {"efel_feature_name": str, "protocol_name": str, "recording_name": str,
-                        "mean": float, "std": float, "efel_settings": dict}
+                        {
+                            "efel_feature_name": "Spikecount",
```

Comment:
Are the keys of this dictionary and that of protocols described anywhere else?

Reply:
They are basically the arguments of `bpem.evaluation.efeature_configuration.EFeatureConfiguration` and `bpem.evaluation.protocol_configuration.ProtocolConfiguration`, so they are also described in these classes docstrings, respectively.

Reply:
Could you mention this here? Users might want to know the details what kind of values can be put in for each key. And if they are not described there then we should describe them there. Thanks!

Reply:
Thanks!